feature-libs/customer-ticketing/root/model/customer-ticketing.model.ts

```diff
@@ -14,6 +14,7 @@ export interface TicketDetails {
   createdAt?: string;
   modifiedAt?: string;
   status?: Status;
+  toStatus?: Status;
```

Comment:
toStatus should be part of the  Event interface, not the TicketDetails :)

Reply:
ohh, let me correct